mods/ts/rules/nod-vehicles.yaml

```diff
@@ -175,6 +175,8 @@ TTNK:
 		RequiresCondition: undeployed
 	RevealOnFire:
 		ArmamentNames: primary, deployed
+	RenderVoxels:
+		Scale: 11.5
```

Comment:
Have you checked how it looks with 12.0?

I've suspected for some time that our default 10.0 is a bit too small, because the voxel-internal scale value that ours probably multiplies with, defaults to 0.083333333~ for all Westwood voxels, so in theory `Scale: 12.0` should give us pretty much the original sizes, disregarding potential rounding errors.